scripts/cmake/vcpkg_configure_meson.cmake

```
@@ -205,7 +205,11 @@ function(z_vcpkg_meson_generate_native_file_config config_type) #https://mesonbu
 
     if(NOT VCPKG_CHAINLOAD_TOOLCHAIN_FILE)
         if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
-            set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${SCRIPTS}/toolchains/windows.cmake")
+            if(VCPKG_TARGET_IS_UWP)
+                set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${SCRIPTS}/toolchains/uwp.cmake")
+            else()
+                set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${SCRIPTS}/toolchains/windows.cmake")
+            endif()
```

```suggestion
            set(VCPKG_CHAINLOAD_TOOLCHAIN_FILE "${SCRIPTS}/toolchains/windows.cmake")
```
revert after https://github.com/microsoft/vcpkg/pull/23846 is merged